packages/core/src/browser/theming.ts

```diff
@@ -110,22 +110,24 @@ export class ThemeService {
         const newTheme = this.getTheme(themeId);
         const oldTheme = this.activeTheme;
         if (oldTheme) {
-            if (oldTheme.id === newTheme.id) {
+            if (oldTheme.id === newTheme?.id) {
                 return;
             }
             oldTheme.deactivate();
         }
-        newTheme.activate();
-        this.activeTheme = newTheme;
-        window.localStorage.setItem('theme', themeId);
-        this.themeChange.fire({
-            newTheme, oldTheme
-        });
+        if (newTheme) {
+            newTheme.activate();
+            this.activeTheme = newTheme;
+            window.localStorage.setItem('theme', themeId);
+            this.themeChange.fire({
+                newTheme, oldTheme
+            });
+        }
```

Comment:
I am not sure about this change; we deactivate the old theme (if any), but we do not necessarily active one, so the app could have no theme. As the author of the original issue, @akosyakov can help with the review.